### PR TITLE
Feat: added home page of quiz app

### PR DESCRIPTION
**Pull request by vivekmehto**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      href="/assets/favicon/gambit-favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="https://gambitui.netlify.app/gambit.min.css" />
```
Comment (tanishq20) [inline index.html]: You can import your component CSS directly in the main CSS file.
By this, you will not have to give a component CSS link in every HTML file.

**Review by tanishq20 — COMMENTED** [review]
Everything looks good. Suggested very small change.

File: index.html (hunk 1 above)
Comment (vivekmehto) [reply index.html]: yes i will change it thank you

**Review by kalyandindu — APPROVED** [review]
I would suggest to make use of few variables for font size and spacing to achieve consistency. Rest all looks fine.

**Pull request merged**